web/src/main/java/com/softserve/osbb/controller/ContractController.java

```diff
@@ -137,22 +160,24 @@ public Contract putContract(@RequestBody Contract contract) {
                 }
             }
         }
+        
         return newContract;
     }
 
     @RequestMapping(value="/{id}",method=RequestMethod.POST)
-    public Contract updateContract(@PathVariable Integer id, @RequestBody Contract contract)
-    {logger.info("Updating contract id:"+id);
+    public Contract updateContract(@PathVariable Integer id, @RequestBody Contract contract) {
+        logger.info("Updating contract id:"+id);
         contract.setContractId(id);
         return contractService.saveAndFlush(contract);
     }
 
     @RequestMapping(value="/",method=RequestMethod.DELETE)
-    public boolean deleteAllContracts(@RequestBody Contract contract)
-    {logger.warn("Deleting all Contracts");
+    public boolean deleteAllContracts(@RequestBody Contract contract) {
+        logger.warn("Deleting all Contracts");
         contractService.deleteAll();
         return true;
```

Comment:
I've go the same question. Problem comes from deleteAll() which is void...
The same problem applies to some other methods.